src/pandas_gpt/completers/litellm.py

```diff
@@ -0,0 +1,14 @@
+from pandas_gpt.completers.openai import OpenAI
+
+__all__ = ["LiteLLM"]
+
+
+class LiteLLM(OpenAI):
+    def run_completion_function(self, **kw):
+        try:
+            import litellm
+        except ImportError:
+            raise Exception(
+                "The package `litellm` could not be found. You can fix this error by running `pip install pandas-gpt[litellm]` or passing a custom `completer` argument."
```

Comment:
nit: If you want to be a pedant like myself about [style guides](https://google.github.io/styleguide/pyguide.html), consider breaking long lines.

Reply:
Might as well! Updated.